cluster/addons/fluentd-elasticsearch/fluentd-es-image/run.sh

```diff
@@ -20,4 +20,4 @@
 # For systems without journald
 mkdir -p /var/log/journal
 
-exec /usr/local/bin/fluentd $FLUENTD_ARGS
+exec /usr/local/bin/fluentd "$FLUENTD_ARGS"
```

Comment:
this string probably _does_ need to be split

Reply:
Don't worry, there's nothing wrong with modifying the detection this way

Reply:
"${FLUENTD_ARGS}" is that ok?

Reply:
that will produce the same result, i'm pretty sure we should just disable the lint for this line since these are arguments and this is actually (hacky) desired behavior